gems/pending/util/miq-process.rb

```diff
@@ -205,6 +205,13 @@ def self.find_pids(cmd)
           pids << pid.to_i
         end
       end
+    when :macosx
+      require 'sys/proctable'
```

Comment:
It does support linux too. I'll look into replacing the the old code with the gem in another PR.